DEPLOYING-FOR-TESTING.md

```diff
@@ -0,0 +1,125 @@
+# Deploying the operator for testing
+
+This section describess testing your local changes of the custom domains operator. 
+First, create a cluster in stage where you plan to deploy the custom domains operator.
+
+
+### Building and pushing image in Quay
+
+To deploy the operator from your branch, you have to create an image in Quay. First, log in to quay and create a folder where you will push the image. In this example I will build the image in quay and push it to my quay account **aliceh** to a folder named **cdo**  with a tag **0**.
+
+```
+docker build . -f build/Dockerfile -t quay.io/ahubenko/cdo:0
+```
+
+```
+docker push quay.io/aliceh/cdo:0 
+```
```

Comment:
I think we have Makefile rules to build and push images (`docker-build` and `docker-push` iirc). Is there any advantage or reason to directly invoke `docker`/`podman` here?

Reply:
Seems like this would require setting env vars which is not obvious.
Would it be all right to add this alternate way to build/push later?